package.json

```diff
@@ -17,7 +17,7 @@
     "test": "test"
   },
   "scripts": {
-    "test": "mocha -t 10000 -r ts-node/register test/**/**/*.spec.ts",
+    "test": "NODE_ENV=test mocha -t 10000 -r ts-node/register test/**/**/*.spec.ts",
```

Comment:
@marianorodriguez perhaps we can add a `test_verbose` version?

Reply:
We can have a "npm run test" command and another "npm run test:silent" or something like that. Do you agree?